### ISSUE TITLE
[Fixed in GH-6052] Firefox: Missing Visual Focus on My Account: Address Book

### ISSUE THREAD

**Issue by developpeurweb** [SAP/cloud-commerce-spartacus-storefront#4330]
Even though the button **Add New Address** has visual focus this is not perceivable enough for accessibility purposes, it should be treated as missing visual focus.

Users with disabilities should be able to switch addresses using the keyboard, currently the options:

- Set as default
- Edit
- Delete

are not keyboard accessible, therefore not receiving focus, thus no visual focus.

![Screen Shot 2019-08-29 at 8 45 57 PM](https://user-images.githubusercontent.com/15113219/63985544-39826080-ca9e-11e9-9b56-0adf98a0f54a.png)

**Comment by Zeyber** [comment]
Card links are now accessible via tab key and focus can be seen.

**Comment by KateChuen** [comment]
Works on chrome.

These do not work:
- Add New Address button: on Safari
- Edit, Delete, Set as Default buttons: on Firefox.

**Comment by KateChuen** [comment]
On chrome & safari, they have the visual focus. But I can't select the buttons: Set as Default, Edit, Delete using the keyboard.

These are missing the visual focus on firefox:
- Edit, Delete, Set as Default buttons. 
- the "Add New Address" focus is grey.

**Comment by developpeurweb** [comment]
**For QA:**

_the "Add New Address" focus is grey._
The grey outline comes from Bootstrap, this is fine for now, it should go away when we remove that framework.

- To test in Safari please refer to [these instructions](https://github.com/SAP/cloud-commerce-spartacus-storefront/issues/5041#issuecomment-558883497).
- This got fixed in ticket GH-6052
- Theme `calydon` should be enabled to see changes.
- Visible focus outline overlapping adjacent ui elements may happen and may not be an issue, if in doubt please open a ticket per element or reach out to UI developer.
- Please separate visible focus issues (CSS: element without outline) from tabbing issues (HTML/TS: element not reachable with keyboard).

![Screen Shot 2020-01-31 at 3 17 00 PM](https://user-images.githubusercontent.com/15113219/73573202-cca23680-4440-11ea-829b-3a0d9988f126.png)

**Comment by KateChuen** [comment]
Tested on dev3.
- Chrome
- Firefox
- Safari

Only issue is when we press ENTER on those "set as default", "edit" and "delete" buttons: #6512
